EventKit/eventkit-cloud
# Refactor provider type filter component to be collapsible

## Description of Improvement

Adds react-collapsible to handle easily creating collapsible components. 
Updates the type filter component in ExportInfo to be a collapsible component.
Adds frontend test to prove collapse functionality works.

Pins react-collapsible component to 2.10 which should be the latest version. Current approved version of dependency doesn't support version of react used by the project.

## How to test

Navigate to second step of create datapack process.
Open 'Sort / Filter'
Verify collapsible type filter component exists
Verify collapse functionality
Verify filter functionality still works
Verify unit tests pass

## Quality Assurance 

The following items have been updated:
 
## Screenshots

<img width="1786" alt="Screen Shot 2022-07-18 at 11 31 50 AM" src="https://user-images.githubusercontent.com/3164631/179547498-17b6c54f-4025-4a80-b7a2-0a9e02f801ed.png">
<img width="1786" alt="Screen Shot 2022-07-18 at 11 31 59 AM" src="https://user-images.githubusercontent.com/3164631/179547534-c6f2ff0e-7907-465c-ae81-7624456973f7.png">


:smile:
